src/jit/unwind.h

```diff
@@ -590,6 +590,11 @@ class UnwindFragmentInfo : public UnwindBase
 public:
     UnwindFragmentInfo(Compiler* comp, emitLocation* emitLoc, bool hasPhantomProlog);
 
+    void Init(Compiler* comp, emitLocation* emitLoc, bool hasPhantomProlog)
+    {
+        UnwindFragmentInfo(comp, emitLoc, hasPhantomProlog);
```

Comment:
This is not correct, it just creates a temporary instance of the UnwindFrameInfo and then throws it away.

Reply:
I'm leaning towards copy-constructor. Is there a better solution?

Reply:
This turned out more work than I was expecting. The cleanest solution is placement new but for some reason windows compiler is refusing placement new.

I tried a couple of experiments but couldn't get far.

I instead renamed constructors with additional arguments to init.